.travis.yml

```diff
@@ -14,11 +14,11 @@ before_install:
   - echo "https://$GH_TOKEN:@github.com" > .git/credentials
   - gem install asciidoctor
 install:
-- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=true
+- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=false
```

Comment:
The intent of this PR is to reduce noise when you're looking at it (ie laptop, stdout).
But one can't look at other files on travis, so this needs to be on stdout for travis (although too long files seem to be a problem as well there).